src/index.ts

```diff
@@ -104,9 +106,43 @@ function isPrimitive(p: Property | Parameter): boolean {
   return !!p.$ref.enum || !p.$ref.properties;
 }
 
-function isDateTime(p: Property | Parameter | null): boolean {
-  if (!p) return false;
-  return p.type === "string" && p.format === "date-time";
+export type XtpTyped = { xtpType: XtpNormalizedType };
+
+function isDateTime(p: XtpTyped): boolean {
```

Comment:
These are helpers to quickly check the type of a node without drilling into the xtpType.kind. Will work on any XtpTyped object.